Dockerfile.worker

```diff
@@ -0,0 +1,15 @@
+FROM node:20-alpine
+
+WORKDIR /app
+
+ENV NODE_ENV production
```

Comment:
I notice we hard code this to `production` in this file and in the main `Dockerfile`.  Can you not run `dev` mode in docker?  That seems a real loss!  Can we easily add that?